src/layouts/HeroSection.tsx

```diff
@@ -15,7 +17,12 @@ const HeroSectionPage = () => {
         </p>
       </div>
       <AboutSection />
+      <div className="mt-44"></div>
+      <ServicesSection />
+      <div className="mt-44"></div>
+      <Testmonials/>
       <div className="min-h-screen"></div>
+      
```

Comment:
and the reason why they are not on the same line as the above one is just take your component and paste in the below of `AboutSection` you will align your logics regarding the results it will give you.